pkg/instrumentation/sdk.go

```diff
@@ -321,3 +328,10 @@ func getIndexOfEnv(envs []corev1.EnvVar, name string) int {
 	}
 	return -1
 }
+
+func moveEnvToListEnd(envs []corev1.EnvVar, idx int) []corev1.EnvVar {
+	envToMove := envs[idx]
+	envs = append(envs[:idx], envs[idx+1:]...)
```

Comment:
there are some corner cases which this implementation does not handle probably. What if the idx equals to envs.lenght?  What if the index is higher than envs.length?